doc/api/stream.md

```diff
@@ -1359,6 +1364,23 @@ The `writable._writev()` method is prefixed with an underscore because it is
 internal to the class that defines it, and should never be called directly by
 user programs.
 
+#### writable.\_final(callback)
+<!-- YAML
+added: REPLACEME
+-->
+
+
+* `callback` {Function} Call this function (optionally with an error
+  argument) when you are done writing any remaining data.
+
+Note: This function **must not** be called directly.  It MAY be implemented
```

Comment:
nit: maybe s/This function/\`_final()\`/ otherwise it sounds like it might be referring to `callback` above?